src/js/actions/type.js

```diff
@@ -96,17 +125,9 @@ define(function (require, exports) {
                     locking.playWithLockOverride(document, layers, setFacePlayObject, typeOptions);
                 });
 
-        var payload = {
-            documentID: document.id,
-            layerIDs: layerIDs,
-            postscript: postscript,
-            family: family,
-            style: style
-        };
-
-        var dispatchPromise = this.dispatchAsync(events.document.TYPE_FACE_CHANGED, payload);
-
-        return Promise.join(dispatchPromise,
+        return Promise.join(function () {
+                    return this.transfer(updatePostScript, document, layers, postscript, family, style);
```

Comment:
Yeah, but it would be cleaner to define them as variables (`transferPromise = this.transfer(...)`) that way you wouldn't have to bind them either and code would be more readable.